hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/SnapshotDiffJob.java

```diff
@@ -15,14 +15,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.hadoop.ozone.om.snapshot;
+package org.apache.hadoop.ozone.om.helpers;
```

Comment:
Should we rename `SnapshotDiffJob` to `SnapshotDiffJobInfo` to be in alignment with other class?

Reply:
I wanted to make the class available to the client and keep diffs to a minimum. We will have to modify around 100 files to make this change.

Reply:
Yeah, it doesn't have to be part of this PR if we agree on it.